rental.rb

```diff
@@ -0,0 +1,13 @@
+require_relative './book'
+
+class Rental
+  attr_accessor :date, :person, :book
+
+  def initialize(date, person, book)
+    @date = date
+    @book = book
+    book.rentals << self
+    @person = person
+    person.rentals << self
+  end
+end
```

Comment:
### Inside `person.rb`:
- [ ] Kindly define rentals as `attr_accessor`.
- [ ] Inside the `initialize` please add `@rentals = []`
- [ ] Please add a method for adding rentals like
```
  def add_rental(book, date)
    Rental.new(date, self, book)
  end
```
to define the relation between person and rental